docs/alexa-apis.rst

```diff
@@ -540,3 +540,216 @@ To send a card requesting user the permission to read/write Alexa lists, you can
+
+    Deletes a reminder
+
+    :param alertToken: Reminder's ID.
+
+    :returns object: A JSON object with the details of reminder's deletion
+
+.. js:class:: ReminderBuilder()
+
+  .. js:method:: setCreatedTime(createdTime)
+
+    Sets created time
+
+    :param createdTime: Reminder's creation time.
+
+    :returns object: A ReminderBuilder object
+
+  .. js:method:: setRequestTime(requestTime)
+
+    Sets request time
+
+    :param requestTime: Reminder's request time.
+
+    :returns object: A ReminderBuilder object
+
+  .. js:method:: setTriggerAbsolute(scheduledTime)
+
+    Sets the reminder trigger as absolute
+
+    :param scheduledTime: Reminder's scheduled time.
+
+    :returns object: A ReminderBuilder object
+
+  .. js:method:: setTriggerRelative(offsetInSeconds)
+
+    Sets the reminder trigger as relative
+
+    :param offsetInSeconds: Reminder's offset in seconds.
+
+    :returns object: A ReminderBuilder object
+
+  .. js:method:: setTimeZoneId(timeZoneId)
+
+    Sets time zone Id
+
+    :param timeZoneId: Reminder's time zone.
+
+    :returns object: A ReminderBuilder object
+
+  .. js:method:: setRecurrenceFreqDaily()
+
+    Sets reminder's recurrence frequence to "DAILY"
+
+    :returns object: A ReminderBuilder object
+
+  .. js:method:: setRecurrenceFreqWeekly()
+
+    Sets reminder's recurrence frequence to "WEEKLY"
+
+    :returns object: A ReminderBuilder object
+
+  .. js:method:: setRecurrenceByDay(recurrenceByDay)
+
+    Sets frequency by day
+
+    :param recurrenceByDay: Array of frequency by day.
+
+    :returns object: A ReminderBuilder object
+
+  .. js:method:: setRecurrenceInterval(interval)
+
+    Sets reminder's interval
+
+    :param interval: Reminder's interval
+
+    :returns object: A ReminderBuilder object
+
+  .. js:method:: addContent(locale, text)
+
+    Sets reminder's content
+
+    :param locale: Reminder's locale
+    :param text: Reminder's text
+
+    :returns object: A ReminderBuilder object
+
+  .. js:method:: enablePushNotification()
+
+    Sets reminder's push notification status to "ENABLED"
+
+    :returns object: A ReminderBuilder object
+
+  .. js:method:: disablePushNotification()
+
+    Sets reminder's push notification status to "DISABLED"
+
+    :returns object: A ReminderBuilder object
+
+With Voxa, you can create, update, delete and get reminders like this:
+
+.. code-block:: javascript
+
+  const { ReminderBuilder } = require("voxa");
+
+  app.onIntent('CreateReminderIntent', async (voxaEvent) => {
+    const reminder = new ReminderBuilder()
+      .setCreatedTime("2018-12-11T14:05:38.811")
+      .setTriggerAbsolute("2018-12-12T12:00:00.000")
+      .setTimeZoneId("America/Denver")
+      .setRecurrenceFreqDaily()
+      .addContent("en-US", "CREATION REMINDER TEST")
+      .enablePushNotification();
+
+    const reminderResponse = await voxaEvent.alexa.reminders.createReminder(reminder);
+
+    voxaEvent.model.reminder = reminderResponse;
```

Comment:
why do we need to add it to the model? @omenocal

Reply:
It's not necessary, it's just an example on how to display the result from the API to a view